test/e2e/apps/daemon_set.go

```diff
@@ -988,6 +988,60 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 		framework.ExpectNoError(err, "failed to locate daemon set %v in namespace %v", testDaemonset.ObjectMeta.Name, ns)
 		framework.Logf("Daemon set %s has a patched status", dsName)
 	})
+
+	ginkgo.Context("Scale subresource", func() {
+		gr := schema.GroupResource{Group: "apps", Resource: "daemonsets"}
+
+		// Create daemonset and ensure it is up
+		ginkgo.BeforeEach(func() {
+			ginkgo.By(fmt.Sprintf("Create a DaemonSet %q.", defaultName))
+			ds, err := c.AppsV1().DaemonSets(ns).Create(context.TODO(), newDaemonSet(defaultName, WebserverImage, defaultLabels), metav1.CreateOptions{})
+			framework.ExpectNoError(err)
+
+			ginkgo.By("Check that daemon pods launch on every node of the cluster.")
+			err = e2edaemonset.CheckDaemonStatus(f, defaultName)
+			framework.ExpectNoError(err)
+			err = wait.PollImmediate(dsRetryPeriod, dsRetryTimeout, checkRunningOnAllNodes(f, ds))
+			framework.ExpectNoError(err)
+		})
+
+		// Delete daemonset
+		ginkgo.AfterEach(func() {
+			ginkgo.By("Clean up.")
+			err := c.AppsV1().DaemonSets(ns).Delete(context.TODO(), defaultName, metav1.DeleteOptions{})
+			framework.ExpectNoError(err)
+		})
+
+		ginkgo.It("can be obtained", func() {
+			scale, err := f.ScalesGetter.Scales(ns).Get(context.TODO(), gr, defaultName, metav1.GetOptions{})
+			framework.ExpectNotEqual(scale, nil, "expected to have non-nil value")
+			framework.ExpectNoError(err, "unexpected error getting daemonsets/scale")
+		})
```

Comment:
can we also check the NotFound error for Get here?

Reply:
done